emcache/base.py

```diff
@@ -327,6 +327,25 @@ async def stats(self, memcached_host_address: MemcachedHostAddress, *args: str)
         Please see a lot of detailed information in the documentation.
         """
 
+    @abstractmethod
+    async def verbosity(
+        self,
+        memcached_host_address: Union[MemcachedHostAddress, MemcachedUnixSocketPath],
+        level: int,
+        *,
+        noreply: bool = False
+    ) -> None:
+        """Set verbosity level in runtime. Verbosity option print STDOUT/STDERR information server.
+        Increase level increase verbose information memcached.
```

Comment:
Better "Increase level of log verbosity for a memcached server" ?

Reply:
Exactly! We are writing documentation to simplify understanding for the client and not to convey exactly the full specifications of the memcached verbosity command